Automattic/jetpack
# Add Carousel for single image

Fixes #375
#### Changes proposed in this Pull Request:
- Single image when linking to itself should not open full-sized image in new window. Instead it should open the image in the same view as it is presented in Carousel gallery.
#### Testing instructions:
- Go to "new post" form
- Add a single image to the post content: it should link to "Media File"
- When image is added instead check the "Text" tab in the editor. There should be a `gallery` shortcode instead of tags a>img
#### Commit message:

Implementation of https://github.com/Automattic/jetpack/issues/375
Initial version without configuration flag - feature is always on.
Replace single image that links to itself (Media File) with a
one-image gallery, so that the site look&feel is preserved when
image is clicked.
